src/ApolloServer.js

```diff
@@ -6,14 +6,20 @@ const { renderPlaygroundPage } = require("@apollographql/graphql-playground-html
 const accept = require("@hapi/accept");
 const moleculerApollo = require("./moleculerApollo");
 
-function send(req, res, statusCode, data, responseType = "application/json") {
+async function send(req, res, statusCode, data, responseType = "application/json") {
```

Comment:
The change to make this async is very concerning.  There are several calls to this method which are not expecting this to return a promise.  There's a very good chance your CI run failures are a result of that.